packer/scripts/centos/openstack.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash -eux
 
-yum -y install cloud-init cloud-utils dracut-modules-growroot cloud-utils-growpart
+packages="cloud-init cloud-utils dracut-modules-growroot cloud-utils-growpart"
+
+if [ $(uname -m)=="ppc64" -o $(uname -m)=="ppc64le" ]
+  then
+    packages="$packages ppc64-diag"
+fi
+
+yum -y install $packages
```

Comment:
I would just move this command directly into the if block above instead of using a variable for it.